src/App.tsx

```diff
@@ -9,43 +9,43 @@ import { LightTheme, DarkTheme } from 'theme.styled';
 import Login from 'components/pages/login';
 import ProtectedRoute from 'components/routes/protected-route';
 import FilmPage from 'components/pages/film-page';
+import Films from 'components/pages/films';
 import 'global.css';
 import Header from 'components/common/header';
 import Profile from 'components/pages/profile';
-
-const Films = React.lazy(() => import('components/pages/films'));
+import { IFilm } from 'interfaces/film.interface';
 
 const App = () => {
   const [darkTheme, setDarkTheme] = useState(false);
-  const [films, setFilms] = useState([]);
-  const [userFilms, setUserFilms] = useState([]);
-  const [nextPage, setNextPage] = useState(1);
-  const [isLogged, setIsLogged] = useState(false);
+  const [films, setFilms] = useState<IFilm[]>();
+  const [userFilms, setUserFilms] = useState<IFilm[]>();
```

Comment:
const [userFilms, setUserFilms] = useState<IFilm[]>([]);
don't forget to provide initial state